azure-stack/hci/manage/manage-syslog-forwarding.md

```diff
@@ -0,0 +1,469 @@
+
+In this configuration, the syslog client in Azure Stack HCI forwards messages to the syslog server over UDP, with no encryption. The client doesn’t verify the identity of the server nor does it provide its own identity to the server for verification.
+
+```azurepowershell
+Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -UseUDP
+```
+
+While UDP with no encryption is the easiest to configure, it doesn’t provide any protection against man-in-the-middle attacks or eavesdropping of messages.
+
+> [!IMPORTANT]
+> Microsoft recommends that you do not use this configuration in production environments.
+
+## Remove syslog forwarding
+
+Run the following command to remove the syslog forwarder configuration and stop the syslog forwarder:
+
+```azurepowershell
+Set-AzSSyslogForwarder -Remove 
+```
+
+## Verify syslog setup
+
+After you successfully connect the syslog client to your syslog server, you will start to receive event notifications. If you don’t see notifications, verify your cluster syslog forwarder configuration by running the following cmdlet:
+
+```azurepowershell
+Get-AzSSyslogForwarder [-Local | -PerNode | -Cluster] 
+```
+
+Each host has its own syslog forwarder agent that uses a local copy of the cluster configuration. They are always expected to be the same as the cluster configuration. You can verify the current configuration on each host by using the following cmdlet:
+
+```azurepowershell
+Get-AzSSyslogForwarder -PerNode 
+```
+
+You can also use the following comdlet to verify the configuration on the host you are connected to:
+
+```azurepowershell
+Get-AzSSyslogForwarder -Local
+```
+
+Cmdlet parameters for the `Get-AzSSyslogForwarder` cmdlet:
+
+|Parameter |Description |Type |Required |
+|----|----|----|----|
+|Local |Show currently used configuration on current host. |Flag |No |
+|PerNode |Show currently used configuration on each host. |Flag |No |
+|Cluster |Show currently global configuration on Azure Stack HCI. This is the default behavior if no parameter is provided. |Flag |No |
+
+## Enable syslog forwarding
+
+```azurepowershell
+Enable-AzSSyslogForwarder [-Force]
+```
+
+Syslog forwarder will be enabled with the stored configuration provided by the last successful `Set-AzSSyslogForwarder` call. The cmdlet will fail if no configuration has been provided using `Set-AzSSyslogForwarder`.
+
+## Disable syslog forwarding
+
+```azurepowershell
+Disable-AzSSyslogForwarder [-Force] 
+```
+
+Parameter for `Enable-AzSSyslogForwarder` and `Disable-AzSSyslogForwarder` cmdlets:
+
+|Parameter |Description |Type |Required |
+|----|----|----|----|
+|Force |If specified, an action plan will always be triggered even if the target state is the same as current. This can be helpful when out-of-band changes need to be reset. |Flag |No |
+
+### [Syslog message schema (appendix)](#tab/syslog-message-schema)
+
```

Comment:
Appendix: Syslog message schema

Reply:
Done > revised tab label to [Syslog message schema]